src/transfers-history/client.ts

```diff
@@ -185,11 +196,17 @@ export class TransfersHistoryClient {
     );
   }
 
-  public getFilledTransfers(depositorAddr: string, limit?: number, offset?: number) {
+  public getFilledTransfers(depositorAddr: TrackableAddress, limit?: number, offset?: number) {
+    if (depositorAddr === "all") {
```

Comment:
this might be cleaner as a new function because you will just have to end up switching something on the frontend, might as well just be a different function rather than address or "all"

Reply:
I started with this thought in mind, but this implied creating separate functions for other things too, for example: stop fetching transfers. For me, passing "all" seemed a cleaner implementation.